R/access-element.R

```diff
@@ -0,0 +1,44 @@
+#' @title Add Access Element
+#' @description The add access element creates a list that includes all the required
+#' elements for the access section of an EML. Calling add_access() will output the 
+#' default public access permissions. To create a non default access add principal 
+#' and permission values into allow_principal and add_permission. 
```

Comment:
To customize the access level provide values for the `allow_principal` and `add_permission` arguments.

Reply:
Added.